Kajabi/sage-lib
# SAGE-1025: Remove Attribute Schema in Doc Site

## Description

Remove Attribute Schema section in Rails doc site.


## Screenshots

|  Before  |  After  |
|--------|--------|
| ![Screen Shot 2022-01-10 at 4 23 35 PM](https://user-images.githubusercontent.com/24237393/148848558-5ad5272d-4614-4a52-8cb6-0d88d8cea841.png) | ![Screen Shot 2022-01-10 at 4 23 00 PM](https://user-images.githubusercontent.com/24237393/148848516-fa0763e9-a681-4d41-bebd-0c8f35b4a7e9.png) |


## Testing in `sage-lib`
1. Navigate to any component
2. Select "Code" tab
3. Observe Attribute Schema is no longer present


## Testing in `kajabi-products`
1. (**N/A**) Sage only. Nothing to verify in `kajabi-products`

## Related
Closes #1025
